projects/step-core/src/lib/modules/table/services/items-per-page.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core';
+import { UserService } from '../../../client/generated';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ItemsPerPageService {
+  private defaultItemsPerPage: Array<number> = [10, 25, 50, 100];
```

Comment:
perhaps it makes sense to denote it as ```readonly``` as well, since its name suggests that it should not be changed